smarts/env/hiway_env.py

```diff
@@ -203,12 +201,12 @@ def step(self, agent_actions):
             observations[agent_id] = agent_spec.observation_adapter(observation)
             infos[agent_id] = agent_spec.info_adapter(observation, reward, info)
 
-        for done in agent_dones.values():
+        for done in dones.values():
```

Comment:
If I am placing `"__sim__"` in `dones` I am unsure how to translate that through the gym interface. I can just remove it since all the agents are set done, I wonder if it should be signalled through `infos` as well.

Reply:
Yeah, I think setting all agents to done should be sufficient for gym.